src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/imports/customimportorder/InputCustomImportOrderNoPackage.java

```diff
@@ -14,16 +14,16 @@
 import java.io.*;
 
 
-import java.util.*; // violation
+import java.util.*; // violation 'Extra separation in import group before 'java.util.*''
 
-import java.util.HashMap; // violation
+import java.util.HashMap; // violation 'Extra separation in import group before .*HashMap''
 import java.util.LinkedList;
 // comments between imports
 
-import javax.net.ServerSocketFactory; // violation
+import javax.net.ServerSocketFactory; // violation 'Extra separation in import group before .*ServerSocketFactory''
```

Comment:
Let's drop name of import from message, it looks too obvious.
Apply to all others.

Reply:
ok so replacing all with `'Extra separation in import group before .*''`

Reply:
done